spec/features/user_spec.rb

```diff
@@ -52,7 +52,7 @@
     end
 
     context 'when user not logged in' do
-      it 'denies access to cards' do
+      it 'denies access to cards_mailer' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.